README.md

```diff
@@ -21,12 +21,12 @@ instance methods, and class methods that are supported.
 
 ## features
 
-- Manipulate binary data like a boss, in all browsers -- even IE6!
+- Manipulate binary data like a boss, in all browsers!
 - Super fast. Backed by Typed Arrays (`Uint8Array`/`ArrayBuffer`, not `Object`)
 - Extremely small bundle size (**5.04KB minified + gzipped**, 35.5KB with comments)
-- Excellent browser support (IE 6+, Chrome 4+, Firefox 3+, Safari 5.1+, Opera 11+, iOS, etc.)
+- Excellent browser support (IE, Edge, Chrome, Firefox, Safari, Opera, iOS, etc.)
```

Comment:
No prob!

Would you like to list the supported IE version by MS or technically by the code?
(it means either IE9 or IE11).

Do you consider Opera and Safari evergreen?